Cargo.toml

```diff
@@ -70,6 +70,9 @@ fast-rng = ["rng", "rand"]
 sha1 = ["sha1_smol"]
 md5 = ["md-5"]
 
+[dependencies]
+diesel = {version = "2.1.0", features = ["sqlite"]}
```

Comment:
Hmm, digging back into this it looks like `diesel` actually already depends on `uuid`, so I don't think we can add it as a dependency here. You might not actually need to do anything in this library to use it with `diesel` besides enable its `uuid` feature. It looks like their support is implemented here: https://github.com/diesel-rs/diesel/blob/a122e7b88a1bc1b427dde34ed36ce3f36fad772e/diesel/src/pg/types/uuid.rs#L13.